simcon_project/simcon_project/settings.py

```diff
@@ -141,8 +142,17 @@
 MEDIA_URL = '/media/'
 LOGIN_REDIRECT_URL = '/redirect-from-login'
 
-# Bootstrap Template for Django Tables 
+# Bootstrap Template for Django Tables
 DJANGO_TABLES2_TEMPLATE = "django_tables2/semantic.html"
 MEDIAFILES_DIRS = [
     os.path.join(BASE_DIR, 'static/media')
 ]
+
+#EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+SENDGID_API_KEY=os.getenv('upxjfdaxtzkmkleq')
```

Comment:
is this our secret key? or public key? this seems sketchy to not be an .env variable

Reply:
This is my email password. . . I will delete it.